### PR TITLE
Remove Condition from getting started pipeline

### PR DESCRIPTION
**Pull request by khrm**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#tests) (if functionality changed/added)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#docs) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commits)
- [x] Release notes block has been filled in or deleted (only if no user facing changes)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```release-note
NONE
```

### CODE REVIEW

File: docs/getting-started/pipeline.yaml [hunk 1]
```diff
@@ -22,11 +21,6 @@ spec:
     - name: build-docker-image
       taskRef:
         name: build-docker-image
-      conditions:
-      - conditionRef: "go-test"
-        resources:
-          - name: source-repo
-            resource: source-repo
```
Comment (savitaashture) [inline docs/getting-started/pipeline.yaml:29]: Can we add replacement of conditions ?

I mean conditions are doing `go test` if we remove it now how do we run `go test` :thinking:

Comment (khrm) [reply docs/getting-started/pipeline.yaml:29]: I removed it as it's not important. We don't need that much focus on pipelines.

Comment (savitaashture) [reply docs/getting-started/pipeline.yaml:29]: Okay 

If require may be we can update/add later

I will merge this PR to unblock CI failure :+1:

**Review by savitaashture — APPROVED** [review]
/lgtm

:+1:

**Pull request merged**